roles/manage_cnp/defaults/main.yml

```diff
@@ -0,0 +1,134 @@
+---
+# defaults file for manage_cnp
+# configuration variable for which
+# task for execute within role
+#cnp_task: "drop-db"
+# Variables
+host: localhost
+# CloudNativePG Namespace
+cnpg_namespace: default
+# CloudNativePG Pod Name
+pod_name: cnp-sandbox-1
+# Roles
+db_role: edb_test_user
+db_role_password: admin
+# Databases
+db_name: edb
+# Tables
+tbl_name: films
+# Schemas
+schema_name: edb_schema
+# Views
+view_name: edb_view_notnullfilms
+# Indexes
+index_name: edb_index_film_name
+# Sequences
+sequence_name: edb_seq_name
+# Procedures
+procedure_name: edb_procedure_insert_film
+# Functions
+function_name: edb_get_films_count
+# Triggers
+trigger_function_name: log_check_filmname_changes
+trigger_name: edb_films_trigger
+# Types
+type_tbl_name: courses
+type_name: edb_type_enum
+
+# SQL Statements
+# Roles
+add_role_sql: |
+     CREATE ROLE {{ db_role }} LOGIN PASSWORD '{{ db_role_password  }}';
+drop_role_sql:
+  DROP ROLE {{ db_role }};
+# Databases
+add_db_sql: |
+  CREATE DATABASE {{ db_name }};
+drop_db_sql: |
+  DROP DATABASE {{ db_name }};
+# Tables
+add_tbl_sql: |
+  CREATE TABLE IF NOT EXISTS {{ tbl_name }}(
+     filmid integer PRIMARY KEY GENERATED BY DEFAULT AS IDENTITY,
+     name   varchar(40) NOT NULL CHECK (name <> '')
+  );
+drop_tbl_sql: |
+  DROP TABLE {{ tbl_name }};
+# Schemas
+add_schema_sql: |
+  CREATE SCHEMA IF NOT EXISTS {{ schema_name }};
+drop_schema_sql:
+  DROP SCHEMA {{ schema_name }};
+# Views
+add_view_sql: |
+  CREATE VIEW {{ view_name }} AS
+    SELECT name FROM {{ tbl_name }} WHERE name IS NOT NULL;
+drop_view_sql:
+  DROP VIEW {{ view_name }};
+# Indexes
+add_index_sql: |
+  CREATE UNIQUE INDEX {{ index_name }} ON
+    {{ tbl_name }} (name);
```

Comment:
Using the pipe ` | ` will add `\n` at line breaks. If you want to remove `\n` line breaks, the `>` folded scalar will turn line breaks into spaces.